docs/explanation/rocks.rst

```diff
@@ -0,0 +1,26 @@
+.. _rocks_explanation:
+
+ROCKs
+=====
+
+ROCKs are container images that are compliant with the `Open Container
+Initiative`_ (OCI) `image format specification <OCI_image_spec_>`_.
+This means that ROCKs are interoperable with other tools that work with
+OCI-compliant images.
+
+A ROCK can be stored in an existing container registry, very much like any
+Docker image, and run in the same way as any other container image.
+For example, Docker can be used to run a ROCK with the usual command line
+syntax:
+
+.. code:: bash
+
+   docker run <rock> ...
+
+Interoperability between ROCKs and other containers also extends to the way
+that container images are built. This enables the use of existing build
+recipes, such as Dockerfiles, with ROCKs. It also opens up the possibility of
```

Comment:
this 2nd sentence

> This enables the use of existing build ￼recipes, such as Dockerfiles, with ROCKs.

is misleading. You can't use Dockerfile with Rockcraft. I see the confusion as a re-write of the old paragraph, which wasn't super clear either. But the point is: even if you have your own Dockerfile and want to keep building Docker images, you can use ROCKs as bases within your Dockerfile, cause they are as valid as any other Docker base image

Reply:
I'll reword it to clarify.

Reply:
ty